lang-haskell/cabal-install/autobuild/build

```diff
@@ -1,7 +1,7 @@
-./bootstrap.sh --sandbox "$SRCDIR"/tempinstall
+abinfo "Building Cabal using existing Cabal ..."
+cabal update
+cabal v2-build --ghc-options="-O2 -fPIC" --prefix="$PKGDIR"/usr --project-file=cabal.project.release cabal-install
 
-mkdir -p "$PKGDIR"/usr/share
-cp -r "$SRCDIR"/tempinstall/bin "$PKGDIR"/usr/
-cp -r "$SRCDIR"/tempinstall/share/man "$PKGDIR"/usr/share/
-
-cp -r "$SRCDIR"/bash-completion "$PKGDIR"/usr/share/
+abinfo "Installing Cabal ..."
+install -dv "$PKGDIR"/usr/bin
+cabal v2-install --install-method=copy --installdir="$PKGDIR"/usr/bin --project-file=cabal.project.release cabal-install
```

Comment:
Please consider splitting with `\`, this would be unreadable on Nano (LOL).

Reply:
Fixed